tests/e2e/test-lint-js.R

```diff
@@ -0,0 +1,13 @@
+rhino::lint_js()
+
+# Create bad scripts and test if formatting returns the expected result
+test_js_path <- fs::path("app", "js", "badStyle.js")
+cat("function sayHello() {console.log('Hello')}; export{sayHello};", file = test_js_path)
+testthat::expect_error(rhino::lint_js())
+rhino::lint_js(fix = TRUE)
+testthat::expect_equal(
+  readLines(test_js_path),
+  "function sayHello() { console.log('Hello'); } export { sayHello };"
+)

```

Comment:
For functions that edit a file, I am using this strategy of creating a file with simple code, bad styling in this case, confirm that the lint returns an error, fix the styling using the appropriate functions, read the contents of the file, use `testthat::expect_equal()` to confirm that the content of the file is as expected.